### PR TITLE
Dependency check task

### PR DESCRIPTION
**Pull request by jack-education**
Build task to identify vulnerable dependencies.

Uses the OWASP Dependency Check CLI tool:
https://owasp.org/www-project-dependency-check/
https://jeremylong.github.io/DependencyCheck/dependency-check-cli/index.html

Sends the results to log analytics for visibility in grafana.

### CODE REVIEW

File: tasks/DependencyCheck/azure-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  displayName: 'PowerShell Script: Remove tests folder'
+- script: tsc
+  workingDirectory: $(System.DefaultWorkingDirectory)/tasks/DependencyCheck/task
+  displayName: 'Command Line Script: tsc'
+- powershell: |
+   $FileName = "$(System.DefaultWorkingDirectory)/tasks/DependencyCheck/task/node_modules/csvtojson/test/data/data#139"
+   if (Test-Path $FileName)
+   {
+     Remove-Item $FileName
+   }
```
Comment (adam230594) [inline tasks/DependencyCheck/azure-pipelines.yml]: Might be good to PR this out of their repo if it affects our build.

Comment (jack-education) [reply tasks/DependencyCheck/azure-pipelines.yml]: Agreed, PR made: https://github.com/Keyang/node-csvtojson/pull/367

**Comment by chelnak** [comment]
I would like to see a demo before it's merged please 👍

**Comment by chelnak** [comment]
Had a run through with @jack-education and am happy for MVS.

@jack-education Does CI update the task version for us?

**Comment by jack-education** [comment]
> Had a run through with @jack-education and am happy for MVS.
> 
> @jack-education Does CI update the task version for us?

@chelnak It doesn't, shall I look to add it as part of MVS?

**Comment by chelnak** [comment]
Maybe not for MVS but it needs to be there asap afterwards.

**Pull request merged**